packages/tenant-process/src/services/tenantService.ts

```diff
@@ -509,4 +564,34 @@ async function assignCertifiedAttribute({
   return updatedTenant;
 }
 
+function assignDeclaredAttribute(
+  targetTenant: Tenant,
```

Comment:
```suggestion
  attributes: TenantAttribute[],
```
You just need the attributes, not the whole tenant